packages/utils/doc/useInventory.md

```diff
@@ -0,0 +1,62 @@
+## Inventory hook
+
+If you are using newer version of react, you can use hook that comes with this package to load inventory and reducers.
+
+```JSX
+import React, { useEffect, useState, Fragment } from 'react';
+import { useStore } from 'react-redux';
+import { useInventory } from '@redhat-cloud-services/frontend-components-utilities/files/useInventory';
+import { getRegistry } from '@redhat-cloud-services/frontend-components-utilities/files/Registry';
+
+const MyCmp = () => {
+    const [ConnectedInventory, setInventory] = useState();
+    const { InventoryTable, newReducers } = useInventory({
+        tableReducer: entitiesReducer,
```

Comment:
Might want to specify from where `entitiesReducer` originates 🤔

Reply:
Edded some more explanantion to the docs and proper import.